nixos/modules/services/web-apps/peertube.nix

```diff
@@ -312,6 +317,7 @@ in {
         ExecStartPre = let preStartScript = pkgs.writeScript "peertube-pre-start.sh" ''
           #!/bin/sh
           umask 077
+          ${lib.optionalString ((cfg.redis.passwordFile != null) || (cfg.database.passwordFile != null)) ''
```

Comment:
I think this will not override the file now if you change the settings.

Reply:
In this variant, the `local.yaml` file is generated if there are passwords for redis and/or postgresql.

Reply:
I know but if you remove the passwords again the old file is still there. So overriding with an empty file like before should be easier and work better.

Reply:
Added this command:
```
test -f /var/lib/peertube/config/local.yaml && rm /var/lib/peertube/config/local.yaml
```

Reply:
I know that this also works but isn't it just easier to override the file with empty contents like before (so remove line 320 and 321)?